### ISSUE TITLE
Add support for better quality artworks

### ISSUE THREAD

**Issue by FatKiwi** [TripleFrequency/AthamePlugin.Deezer#37]
I discussed this in the same Discord group you're in.

2000x2000 artworks + JPG quality 94.
Some artworks are available in 2000x2000. The default JPG quality is 80 at the moment for all artworks (which has some degradation). Quality 94 is more than enough and each step afterwards makes the filesize big.

Example URL: 
2000x2000-000000-94-0-0.jpg

What do you think about it?

**Comment by Freetekno** [comment]
if the use of so images/cover/0000000000000000000000/9999x9999.png and to replace jpg to png, we will get original image.

**Comment by FatKiwi** [comment]
Not exactly. The 9999x9999 reverts to 1200x1200, whereas 2000x2000 does not.
And PNG is larger by 1.5x than Quality 100 JPG, and the filesizes are around 4-6MB. It's unsuitable.
